pandas/tests/indexing/test_floats.py

```diff
@@ -498,122 +498,72 @@ def test_floating_index_doc_example(self):
         assert s.loc[3] == 2
         assert s.iloc[3] == 3
 
-    def test_floating_misc(self):
+    @pytest.mark.parametrize("indexer_sl", [tm.getitem, tm.loc])
```

Comment:
you dont need to parametrize since indexer_sl is a fixture

Reply:
Thx, did not get that :(